opentelemetry-zipkin/Cargo.toml

```diff
@@ -22,7 +22,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
 default = ["reqwest-blocking-client", "reqwest/native-tls"]
-reqwest-blocking-client = ["reqwest/blocking", "opentelemetry-http/reqwest"]
+reqwest-blocking-client = ["reqwest/blocking", "opentelemetry-http/reqwest-blocking"]
 reqwest-client = ["reqwest", "opentelemetry-http/reqwest"]
 reqwest-rustls = ["reqwest", "reqwest/rustls-tls-native-roots"]
```

Comment:
should we expose the feature `reqwest-rustls-webpki-roots` here too?

Reply:
lets tackle it in a follow up..this crate's dependencies/features require a proper review, better handle it in own PR.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/3113